sympy/assumptions/tests/test_query.py

```diff
@@ -2216,6 +2216,10 @@ def test_issue_9636():
     assert ask(Q.odd(3.0)) is False
 
 
+def test_issue_12168():
```

Comment:
I wouldn't create a separate functions, just add the test for `ask(Q.rational(math.pi))` with the others above with a comment "# test for issue #12168".